woocommerce/woocommerce
# Remove red badge on extensions menu when the site is not connected

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





- Removes the red badge under in-app My Subscriptions and Extensions menu when the site is not connected.
- Update the icon shown on disconnected state for in-app my subscriptions.
- Increase the width of the element showing in-app connect message.

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start your development environment. `pnpm -- wp-env start`
2. Build assets `pnpm --filter='@woocommerce/plugin-woocommerce' build`
3. Make sure site is not connected.
4. There should not be a bubble shown on Extensions menu or on the my subscriptions tab when the site is not connected.
5. If you are seeing the `sales` bubble on Extensions menu item, that can be removed by clearing the transient.
```
pnpm -- wp-env run cli bash
wp shell
delete_transient('woocommerce_marketplace_promotions');
```
6. Visit [in-app My Subscriptions] (http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions).
7. The, icon shown when the site is not connected should be updated to a new one. The width of the not connected message and it's text color is also updated.

<img width="2043" alt="Screenshot 2024-04-23 at 13 49 03" src="https://github.com/woocommerce/woocommerce/assets/1531403/d4380020-e350-48f0-bfe1-aa982d6d1790">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Remove the red badge shown on extensions menu item when the site is not connected to WooCommerce.com

#### Comment 

</details>
